bindgen/spec.yml

```diff
@@ -1208,13 +1209,13 @@ classes:
   SyncUser:
     cppName: SyncUser
     sharedPtrWrapped: SharedSyncUser
+    properties:
+      is_logged_in: bool
 
   User:
-    base: SyncUser
```

Comment:
I removed the declaration of this inheritance, because the SDK emits `class User extends SyncUser` and I saw assertion failures from the `User` being passed as a `SyncUser` into the binding, which just interpreted the shared pointer without performing the downcast.